apps/expo/app/(app)/(drawer)/(tabs)/(stack)/profile/[id].tsx

```diff
@@ -4,25 +4,22 @@ import { StyleSheet, Text, View, Platform } from 'react-native';
 import ProfileContainer from 'app/screens/user/ProfileContainer';
 import Head from 'expo-router/head';
 import { useProfileId } from 'app/hooks/user';
-import { useProfile } from 'app/hooks/user';
 
 const Profile = () => {
   const [id] = useProfileId();
-  const { user } = useProfile(id);
-  const userRealName = user?.name ?? null;
 
   return (
     <>
       {Platform.OS === 'web' && (
         <Head>
-          <title>{`${userRealName}'s Profile`}</title>
-          <meta name="description" content={`${userRealName}'s Profile`} />
+          <title>{`${id}'s Profile`}</title>
+          <meta name="description" content={`${id}'s Profile`} />
```

Comment:
This is needs to be reverted, and the ones above and below as well.

Reply:
i reverted it